webpack.config.js

```diff
@@ -165,18 +166,21 @@ if (!__TEST__) {
 // ------------------------------------
 // Rules
 // ------------------------------------
+const jsLoaders = [{
+  loader: 'babel-loader',
+  options: {
+    cacheDirectory: true,
+  },
+}]
+if (__DEV__) jsLoaders.push('react-hot-loader/webpack')
+
 webpackConfig.module.rules = [...webpackConfig.module.rules, {
   //
   // Babel
   //
   test: /\.js$/,
   exclude: /node_modules/,
-  use: {
-    loader: 'babel-loader',
-    options: {
-      cacheDirectory: true,
-    },
-  },
+  loaders: jsLoaders,
```

Comment:
@levithomason Thanks for your eyes, [docs](https://github.com/gaearon/react-hot-loader/tree/master/docs#migration-to-30) sent me the wrong way